src/classes/storage.ts

```diff
@@ -1,36 +1,82 @@
-import { StopGroup, StopCluster, StopGroupGpuid, StopClusterGpuid } from '../types';
+import {
+  GroundPlacesObject,
+  StopGroup,
+  StopCluster,
+  StopGroupGpuid,
+  StopClusterGpuid,
+  GroundPlaceType,
+  Gpuid,
+  StopGroupProperties,
+  StopClusterProperties,
+} from '../types';
 
 /**
  * @description Manipulate GroundPlaces.
  */
 export class Storage {
+  private readonly groundPlaces: GroundPlacesObject;
+
+  constructor(groundPlacesFile: string) {
+    this.groundPlaces = this.readJSONFile(groundPlacesFile);
+  }
+
+  /**
+   * @description Parse the JSON File that have all the ground places.
+   * @param {string} groundPlacesFile - The JSON File to parse.
+   * @returns {GroundPlacesArray}
+   */
+  public readJSONFile(groundPlacesFile: string): GroundPlacesObject {
+    return JSON.parse(groundPlacesFile);
+  }
+
+  /**
+   * @description Getter to retrieve the Ground places.
+   * @returns {GroundPlacesObject}
+   */
+  public getGroundPlaces(): GroundPlacesObject {
+    return this.groundPlaces;
+  }
+
   /**
    * @description Returns the stopGroup identified by its Gpuid.
    * @param {StopGroupGpuid} stopGroupGpuid - Ground Place unique identifier of the StopGroup to find.
-   * @returns {StopGroup|undefined}
+   * @returns {StopGroup}
    */
-  // @ts-ignore
-  public getStopGroupByGpuid(stopGroupGpuid: StopGroupGpuid): StopGroup | Error {}
+  public getStopGroupByGpuid(stopGroupGpuid: StopGroupGpuid): StopGroup {
+    const groundPlace = this.groundPlaces[stopGroupGpuid];
```

Comment:
typing missing

Reply:
why not just :
`return this.groundPlaces[stopGroupGpuid]`
catching and throwing errors well defined ?